### PR TITLE
docs(v2): Add READMEs to v2 packages

### PR DESCRIPTION
**Pull request by RDIL**
Signed-off-by: Reece Dunham <me@rdil.rocks>

## Motivation

Adds basic readmes to each of the v2 packages that don't have one.

I didn't put that much content in them, just relevant docs links, because I don't have hours to spare writing all this.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

n/a

## Related PRs

n/a

### CODE REVIEW

File: packages/docusaurus-remark-plugin-npm2yarn/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@docusaurus/remark-plugin-npm2yarn",
   "version": "2.0.0-alpha.70",
-  "description": "Remark plugin for convert npm commands to yarn as tabs",
+  "description": "Remark plugin for convert npm commands to Yarn commands as tabs.",
```
Comment (RDIL) [inline packages/docusaurus-remark-plugin-npm2yarn/package.json:4]:
```suggestion
  "description": "Remark plugin for converting npm commands to Yarn commands as tabs.",
```

**Review by lex111 — APPROVED** [review]

File: packages/docusaurus-plugin-sitemap/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# `@docusaurus/plugin-sitemap`
+
+Simple sitemap generation plugin for Docusaurus.
+
+## Usage
+
+See [`@docusaurus/plugin-sitemap` documentation](https://v2.docusaurus.io/docs/api/plugins/@docusaurus/plugin-sitemap).
```
Comment (slorber) [inline packages/docusaurus-plugin-sitemap/README.md:7]:
```suggestion
See [plugin-sitemap documentation](https://v2.docusaurus.io/docs/api/plugins/@docusaurus/plugin-sitemap).
```

for consistency

File: packages/docusaurus/README.md [hunk 3]
```diff
@@ -1 +1,74 @@
...
+
+## Special thanks
+
+[![BrowserStack logo](./admin/img/browserstack-logo.png)](http://www.browserstack.com/)
+
+[BrowserStack](http://www.browserstack.com/) supports us with [free access for open source](https://www.browserstack.com/open-source).
+
+[![Rocket Validator logo](./admin/img/rocketvalidator-logo.png)](https://rocketvalidator.com/)
+
+[Rocket Validator](https://rocketvalidator.com/) helps us find HTML markup or accessibility issues.
```
Comment (slorber) [inline packages/docusaurus/README.md:74]:
```suggestion

```
Lets keep it empty for now as it's a duplicate of the homepage and many links are to v1, we'll need to update this soon anyway

**Comment by slorber** [comment]
Thanks, LGTM 👍 

Removed the Docusaurus readme content as it's mostly v1 links there and we can't simply copy the home readme.

We'll rewrite the home readme very soon anyway

**Pull request merged**